apisix/admin/init.lua

```diff
@@ -47,8 +47,10 @@ local resources = {
     consumers       = require("apisix.admin.consumers"),
     schema          = require("apisix.admin.schema"),
     ssl             = require("apisix.admin.ssl"),
```

Comment:
I think we should delete `ssl`, all right?

Reply:
That's a break change, if so, we need to do a lot of changes to test cases and docs, which can cause conflicts when next merge to master?

Reply:
We can first implement the function to update test cases, documents and uri on master after next merge to master?